### ISSUE TITLE
Does this code work

### ISSUE THREAD

**Issue by krishnakanthpps** [OSDG-IIITH/share-a-cab#1]
Hi.. Does this code work and can I fork it and develop my own version?

**Comment by hharchani** [comment]
Yeah this code works, but it have many bugs. We have a new software ready to completely replace this. We'll be updating that soon. You're welcome to fork any of the two and hack upon it.
